src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -108,7 +108,7 @@ class BaseOptionsSelector extends Component {
             focusedIndex: newFocusedIndex,
         }, () => {
             // If we just toggled an option on a multi-selection page, scroll to top
```

Comment:
Should we update this comment perhaps?

Reply:
Ahh yes, we should add more context about the changes.

@Ollyws Can you update the comment there? Thanks!